.github/ci-scripts/upload_wheel_to_s3.py

```diff
@@ -0,0 +1,39 @@
+import sys
+from pathlib import Path
+
+import boto3
+import constants
+import wheellib
+
+if __name__ == "__main__":
+    commit_hash = sys.argv[1]
+    platform_substring = sys.argv[2]
```

Comment:
nit: prefer argparse, and use a choice instead

Reply:
Updated.